web/packages/teleport/src/Discover/ConnectMyComputer/SetupConnect/SetupConnect.story.tsx

```diff
@@ -105,11 +105,11 @@ export const PollingSuccess = () => (
 PollingSuccess.parameters = {
   msw: {
     handlers: [
-      rest.get(cfg.api.nodesPath, (req, res, ctx) => {
-        return res.once(ctx.json({ items: [] }));
```

Comment:
`PollingSuccess` was not the only story that was supposed to use `res.once`. AFAIK, at the time of the upgrade to v7, all of the uses of `res.once` in teleport.e were unnecessary (see https://github.com/gravitational/teleport.e/pull/3256/commits/5d0367d6cceedc71a62f7fe425873820ca6df1c0), but there was a couple of places in OSS that actually do needed it.

Could you go through them on master and add `{ once: true }` where necessary? I found that just going to the `web` dir and grepping for `res.once` is good enough.